src/main/io/displayport_crsf.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "pg/pg.h"
+#include "drivers/display.h"
+
+#define CRSF_DISPLAY_PORT_ROWS_MAX 8
+#define CRSF_DISPLAY_PORT_COLS_MAX 32
+
+typedef struct crsfDisplayPortRow_s {
+    char data[CRSF_DISPLAY_PORT_COLS_MAX];
+    bool pendingTransmit;
+} crsfDisplayPortRow_t;
+
+typedef struct crsfDisplayPortScreen_s {
+    crsfDisplayPortRow_t rows[CRSF_DISPLAY_PORT_ROWS_MAX];
+    bool reset;
+} crsfDisplayPortScreen_t;
+
+PG_DECLARE(displayPortProfile_t, displayPortProfileCrsf);
```

Comment:
In terms of maintainability, it would be better if this is right after the declaration of `displayPortProfile_t`, to make it obvious that this is used in this parameter group. Or else add a comment in place of it stating that fact.

Reply:
Adding the declaration in `drivers/display.h` caused various compilation errors due to it's widespread inclusion throughout the code base.  Provided a comment instead.

Reply:
Yeah, would probably have had to be made conditional. Comment is a better option - we just need to be able to keep track of what structs fall under the constraints that apply to structs used in parameter groups.